tvb_framework/tvb/core/neocom/_h5loader.py

```diff
@@ -268,8 +268,8 @@ def load_ht_function(sub_gid, traited_attr):
 
 class ViewModelLoader(DirLoader):
```

Comment:
Do I understand correctly that `ViewModelLoader` as well as the new `DtLoader` are somehow similar with the previous `TVBLoader`, the main difference being that the later uses the relational DB for some steps, while the first 2 are solely relying on the disk structure ?
Maybe this aspect can be documented on the classes a bit better.

Reply:
Yes, that is one difference. I will document it, thanks for the suggestion.
TVBLoader is also based on the TVB storage folder structure. The other 2 write in a directory provided as input.

Reply:
Thanks for the details. In the future perhaps we can get rid of the prev TVBLoader all together, as part of our tvb-storage refactor

Reply:
Please check whether the docs are informative enough

Reply:
very informative now. Thank you!